src/OrbitGl/TimeGraph.cpp

```diff
@@ -245,6 +245,10 @@ double TimeGraph::GetTime(double ratio) const {
 }
 
 void TimeGraph::ProcessTimer(const TimerInfo& timer_info, const FunctionInfo* function) {
+  if (timer_info.start() < capture_min_timestamp_) {
```

Comment:
Maybe a stupid question because I'm don't have too much experience with Google C++ Style, but using "min" function here (and "max" after) isn't clearer? (at least less lines of code)

Reply:
done. - this code was modified to use min/max.